addons/binding/org.openhab.binding.sleepiq/OSGI-INF/SleepIQHandlerFactory.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2014-2016 by the respective copyright holders.
```

Comment:
I will push updates shortly with this fixed for all files. Static analysis shows my license headers are all out of date.